packages/quantic/force-app/main/default/labels/CustomLabels.labels-meta.xml

```diff
@@ -147,6 +147,34 @@
         <protected>false</protected>
         <shortDescription>Next</shortDescription>
     </labels>
+    <labels>
+        <fullName>quantic_PreviousPage</fullName>
+        <value>Previous Page</value>
+        <language>en_US</language>
+        <protected>false</protected>
+        <shortDescription>Previous page</shortDescription>
+    </labels>
+    <labels>
+        <fullName>quantic_NextPage</fullName>
+        <value>Next Page</value>
+        <language>en_US</language>
+        <protected>false</protected>
+        <shortDescription>Next page</shortDescription>
+    </labels>
+    <labels>
+        <fullName>quantic_GoToPage</fullName>
+        <value>Go to page {{0}}</value>
+        <language>en_US</language>
+        <protected>false</protected>
+        <shortDescription>Go to page ...</shortDescription>
+    </labels>
+    <labels>
+        <fullName>quantic_ShowNResultsPerPage</fullName>
+        <value>Show {{0}} number of results per page</value>
+        <language>en_US</language>
+        <protected>false</protected>
+        <shortDescription>Show N results per page.</shortDescription>
```

Comment:
French translation: Afficher <nombre> résultats par page.

<nombre> will be replaced by a number, right? E.g.,  "Afficher 5 resultats par page."